src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -5,43 +5,75 @@
  */
 
 import React from 'react';
+import noop from 'lodash/noop';
 import API from '../../../api';
 import VersionsSidebar from './VersionsSidebar';
 import { withAPIContext } from '../../common/api-context';
 
 type Props = {
     api: API,
     fileId: string,
+    onVersionChange: (versionId?: string) => void,
+    parentPath: string,
+    versionId?: string,
 };
 
 type State = {
     error?: string,
-    versions: Array<any>,
+    isLoading: boolean,
+    versions: Array<BoxItemVersion>,
 };
 
 class VersionsSidebarContainer extends React.Component<Props, State> {
     props: Props;
 
     state: State = {
+        isLoading: true,
         versions: [],
     };
 
+    static defaultProps = {
+        onVersionChange: noop,
+    };
+
     componentDidMount() {
         const { api, fileId } = this.props;
 
         api.getVersionsAPI(false).getVersions(fileId, this.handleFetchVersionsSuccess, this.handleFetchVersionsError);
```

Comment:
Do we plan on showing the current version in the version history sidebar? If there is only v1, would this sidebar still show the v1?

Reply:
The data layer on this stuff is still pretty basic at the moment. The idea is to only show version history if there's more than one version, though.

Reply:
From playing around with the current webapp UI, it does looks like even with V1 we can open the version history modal, so as long as this approach is an intentional change instead of an unexpected regression we should be good

Reply:
Good point. Will validate with product before I introduce that logic.